sms/dlr-flask.py

```diff
@@ -4,7 +4,7 @@
 app = Flask(__name__)
 
 @app.route('/webhooks/delivery-receipt', methods=['GET', 'POST'])
-def delivery_receipt():
+def inbound_sms():
```

Comment:
Not correct. This is the delivery receipt webhook, why are you changing it to inbound sms?

Reply:
Just checked. I did the change other way around. Making this fix now.